programs/locker/src/instructions/v2/cancel_vesting_escrow.rs

```diff
@@ -0,0 +1,126 @@
+use anchor_spl::memo::Memo;
+use anchor_spl::token_interface::{
+    close_account, CloseAccount, Mint, TokenAccount, TokenInterface,
+};
+
+use crate::constants::transfer_memo;
+use crate::safe_math::SafeMath;
+use crate::util::{transfer_to_user_v2, MemoTransferContext};
+use crate::*;
+
+/// Accounts for [locker::cancel_vesting_escrow].
+#[derive(Accounts)]
+#[event_cpi]
+pub struct CancelVestingEscrowV2<'info> {
```

Comment:
why not wrap a function for cancel vesting escrow?

Reply:
I don't think we need it anymore since we remove the v1 endpoint.